src/MorganStanley.ComposeUI.MainApp/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using Avalonia;
+using System;
+
+namespace MorganStanley.ComposeUI.MainApp
+{
+    internal class Program
```

Comment:
What is this project, why is it directly under `src`? Is it the Avalonia-based shell? If so, it should be in something like `src/avalonia-shell` in its own solution.

Reply:
Yes, let us call it MorganStanley.ComposeUI.AvaloniaShell!

Reply:
Yes, I called it AvaloniaShellApp